apps/src/blockly/addons/shortcut.js

```diff
@@ -22,9 +22,7 @@ const blockCopyToStorageShortcut = function() {
       // which may beep or otherwise indicate
       // an error due to the lack of a selection.
       e.preventDefault();
-      const blockText = Blockly.Xml.domToPrettyText(
-        Blockly.Xml.blockToDom(Blockly.selected)
-      );
+      const blockText = JSON.stringify(Blockly.selected.toCopyData().saveInfo);
```

Comment:
small nit: you use a different local variable name here vs. in contextMenu to store copied data (blockText vs copyData)